problem_set_1.py

```diff
@@ -91,3 +128,20 @@
 
 # Write a function called `gcd` that takes two arguments and returns the greatest
 # common divisor using the instructions above.
+def gcd(a, b):
+    d = 0
```

Comment:
Also awesome. This could be a fun place to play with ternary operators https://book.pythontips.com/en/latest/ternary_operators.html

Reply:
An attempt was made!...This is another one I might have to come back to though after reading up a bit more on ternary operators. I tried to skate by by just quickly skimming the link you provided as well as a few others, but couldn't quite puzzle out how to get it to work. FWIW, my prior attempt before this one (which also failed to pass the tests) had line 166 instead read:

`a = a / 2 if is_even(a) else (a - b) / 2`
`b = b / 2 if is_even(b) else (b - a) / 2`